src/header&footer/header&footer.scss

```diff
@@ -0,0 +1,28 @@
+.appheader {
+  background-color: teal;
+  height: 55px;
+  display: flex;
+  align-items: center;
+  position: sticky;
+  top: 0;
+  h3 {
+    margin: 0;
+    padding-left: 90px;
+    color: rgb(221, 170, 17);
+  }
+}
+
+.appfooter {
+  background-color: teal;
+  height: 55px;
+  display: flex;
+  align-items: center;
+  position: fixed;
+  bottom: 0;
+  width: 100%;
+  h3 {
+    margin: 0;
+    padding-left: 90px;
+    color: rgb(221, 170, 17);
+  }
```

Comment:
since you're repeating those styles why don't you think of a way to only have them in one place